src/org/thoughtcrime/securesms/connect/DcEventCenter.java

```diff
@@ -18,7 +18,9 @@
 
 public class DcEventCenter {
     private @NonNull final Hashtable<Integer, ArrayList<DcEventDelegate>> allObservers = new Hashtable<>();
```

Comment:
`allObservers` is misleading now, this should be `currentAccountObservers` now

Reply:
yes I agree, didn't change it initially afraid of doing too much unrelated changes that might make reviewing harder, but now that I see you agree will do it :+1: